docs/source/conf.py

```diff
@@ -51,6 +52,10 @@
 # a list of builtin themes.
 #
 html_theme = 'sphinx_rtd_theme'
+html_logo = '_static/piq_logo.png'
+html_theme_options = {
+    'logo_only': True,
+}
```

Comment:
Option 1 - light-grey background 
```suggestion
html_theme_options = {
    'logo_only': True,
    'style_nav_header_background': '#FAFAFA',
}
```